wlroots/ffi_build.py

```diff
@@ -479,6 +479,71 @@ def has_xwayland() -> bool:
     struct wl_display *display);
 """
 
+# types/wlr_idle.h
+CDEF += """
+struct wlr_idle {
+    struct wl_global *global;
+    struct wl_list idle_timers; // wlr_idle_timeout::link
+    struct wl_event_loop *event_loop;
+    bool enabled;
+
+    struct wl_listener display_destroy;
+    struct {
+        struct wl_signal activity_notify;
+        struct wl_signal destroy;
+    } events;
+
+    void *data;
```

Comment:
Can you add `...;` to the end of this struct and the one below? That enables some extra consistency checking to be done when the cffi compile step happens.

Reply:
Done, I also added them to the output_power_management structs from my previous PR